### PR TITLE
Backports 0.20 pr5

### PR DESCRIPTION
**Pull request by Munkybooty**

### CODE REVIEW

**Comment by PastaPastaPasta** [comment]
lint-python fails

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]
lint-python fails

**Comment by Munkybooty** [comment]
Dropping #18407 as it is #4829

**Comment by Munkybooty** [comment]
I'm unsure of how to go about fixing `mempool_accept.py`

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: src/net_processing.cpp [hunk 1]
```diff
@@ -4858,7 +4860,7 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
 
                             auto ret = mapRelay.insert(std::make_pair(hash, std::move(txinfo.tx)));
                             if (ret.second) {
-                                vRelayExpiration.push_back(std::make_pair(nNow + 15 * 60 * 1000000, ret.first));
+                                vRelayExpiration.push_back(std::make_pair(nNow + std::chrono::microseconds{RELAY_TX_CACHE_TIME}.count(), ret.first));
```
[the remark this answers is not in the document]
Comment (Munkybooty) [reply src/net_processing.cpp:4868]: I had attempted to add the extra changes, however i kept encountering errors. I may need a little help in regards to this specific backport.

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
pls see 14f05d8bb22ee90abeb28f4c75d2bebfe137a49f + e9c1ea674ebf149af391cd6c872bebb77800c4e4 to fix 16851 and  561ece03e910d6ab893be3eb47950df64de7d966 + 7b4d2d695cf053d4b07b361f8d424565f999588c to fix 17843

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK

**Review by UdjinM6 — APPROVED** [review]
utACK

**Comment by UdjinM6** [comment]
looks good to merge after #4896 (will have to drop 537f275 (16908) and squash c5adfa7 into 0a8e213 (16851) on rebase)

**Review by UdjinM6 — APPROVED** [review]
utACK

**Comment by UdjinM6** [comment]
@PastaPastaPasta

**Review by UdjinM6 — APPROVED** [review]
re-utACK

File: src/wallet/rpcwallet.cpp [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2010 Satoshi Nakamoto
-// Copyright (c) 2009-2018 The Bitcoin Core developers
+// Copyright (c) 2009-2019 The Bitcoin Core developers
```
Comment (PastaPastaPasta) [inline src/wallet/rpcwallet.cpp:2]: 15930 needs release notes please!

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit however, a subsequent PR that adds release notes for the added RPC should be completed @Munkybooty

**Comment by thephez** [comment]
@Munkybooty Can you update the PR title to include the conventional commit `backport: ` prefix?

**Pull request merged**